meson.build

```diff
@@ -35,3 +35,11 @@ endif
 subdir('data')
 subdir('doc')
 subdir('profile.d')
+
+#mytarget = custom_target('toolbox-rs',
```

Comment:
I don't know meson, tried only briefly to figure out how to glue this together but we can just stick it in the spec file or wrap it in a `Makefile`.